src/clip.sol

```diff
@@ -297,6 +304,14 @@ contract Clipper {
 
     // --- Shutdown ---
 
+    function stop() external auth isStopped(false) {
+        stopped = true;
+    }
+
+    function start() external auth isStopped(true) {
+        stopped = false;
+    }
+
```

Comment:
I'm debating with myself whether these need `isStopped` modifiers... I guess it could help with a caller making a typo-like error (e.g. calling `stop` when the meant `start`), but it could also cause an unexpected revert if someone expects `stop` and `start` calls to be idempotent. For comparison, `ds-stop` doesn't have this check:
https://github.com/dapphub/ds-stop/blob/master/src/stop.sol#L30

Reply:
Any considerations I'm missing here?

Reply:
Yeah I do not feel strongly here

Reply:
I'd remove the modifiers in these functions.